backend/case/src/main/kotlin/com/ritense/case/service/validations/TaskListColumnValidationUtils.kt

```
@@ -85,17 +85,10 @@ open class TaskListColumnValidationUtils(
-            if (!listOf("createTime", "name", "assignee", "dueDate", "assignedTeamTitle").contains(pathWithoutPrefix)) {
-                throw InvalidListColumnException("\"${pathWithoutPrefix}\" is not an option for the task: prefix.", Status.BAD_REQUEST)
-            }
-        } else {
-            try {
-                valueResolverService.validateValues(caseDefinitionName, listOf(path))
-            } catch (ex: Exception) {
-                throw InvalidListColumnException(ex.message, Status.BAD_REQUEST)
-            }
+        try {
+            valueResolverService.validateValues(caseDefinitionName, listOf(path))
+        } catch (ex: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)